src/webui/www/private/views/preferences.html

```diff
@@ -106,6 +106,10 @@
             <input type="checkbox" id="displayFullURLTrackerColumn">
             <label for="displayFullURLTrackerColumn">QBT_TR(Display full announce URL in the Tracker column)QBT_TR[CONTEXT=OptionsDialog]</label>
         </div>
+        <div class="formRow" style="margin-bottom: 3px;">
+            <input type="checkbox" id="useVirtualListColumn">
+            <label for="useVirtualListColumn">QBT_TR(Optimize list performance (experimental))QBT_TR[CONTEXT=OptionsDialog]</label>
```

Comment:
```suggestion
            <label for="useVirtualList">QBT_TR(Enable virtual list for tables (experimental))QBT_TR[CONTEXT=OptionsDialog]</label>
```
IMO "Optimize list performance" is too abstract. User won't able to know what it is referring to. I used "tables" but I'm not sure it is correct. You'll need to change it if it isn't accurate.
Also, `Column` shouldn't be in the `id`.

Reply:
>"Enable optimized table rendering"

👍 I think this is good.